### PR TITLE
Add pytest into the mix instead of nose

### PR DESCRIPTION
**Pull request by ndanielsen**

### CODE REVIEW

**Comment by ndanielsen** [comment]
@bbengfort ready to merge this

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ clean:
 
 # Targets for testing
 test:
-	$(PYTHON_BIN)/nosetests -v --with-coverage --cover-package=$(PROJECT) --cover-inclusive --cover-erase tests
+	$(PYTHON_BIN)/py.test --cov=$(PROJECT) tests/
```
Comment (bbengfort) [inline Makefile]: I think that we should [#dropthedot](https://twitter.com/davehunt82/status/756080181085872129) given that `py.test` is deprecated.

**Comment by bbengfort** [comment]
@ndanielsen let me know if you want me to merge or if you want to update based on my comments.

**Comment by bbengfort** [comment]
@ndanielsen bump -- should I merge?

**Comment by ndanielsen** [comment]
@bbengfort go ahead, I'll handle `setup.py and setup.cfg` soonish

**Comment by bbengfort** [comment]
@ndanielsen cool - I'll see you there!

**Pull request merged**